extensions/srv-records-tcp-3.2

```diff
@@ -0,0 +1,52 @@
+Using SRV records with IRC (TCP protocol)
+=========================================
+
+Copyirhgt (c) 2014 Patrick Godschalk <argure@argure.nl>
```

Comment:
Just happen to be reading this and noticed there's a typo in copyright. Cool stuff though! :)